Sources/PackageLoading/ToolsVersionLoader.swift

```diff
@@ -36,15 +36,15 @@ public class ToolsVersionLoader: ToolsVersionLoaderProtocol {
         public var description: String {
             switch self {
             case .malformed(let versionSpecifier, let file):
-                let file = file.appending(component: ToolsVersion.toolsVersionFileName)
+                let file = Manifest.versionSpecificManifest(path: file)
                 return "The version specifier '\(versionSpecifier)' in '\(file.asString)' is not valid"
             }
         }
     }
 
     public func load(at path: AbsolutePath, fileSystem: FileSystem) throws -> ToolsVersion {
         // The file which contains the tools version.
-        let file = path.appending(component: ToolsVersion.toolsVersionFileName)
+        let file = Manifest.versionSpecificManifest(path: path, fileSystem: fileSystem)
```

Comment:
It is a little unfortunate that we will compute this twice...maybe we should lift it out and let clients handle it. But this is fine for now.